### PR TITLE
Add standalone package.json to contracts folder

### PR DESCRIPTION
**Pull request by Pana**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@conflux-dev/contracts",
+  "name": "@confluxfans/solidity",
```
Comment (boqiu) [inline package.json:2]: How about to keep the same name convention with openzeppelin? E.g. `@confluxfans/contracts`, which is used in the package.json

**Review by wangdayong228 — APPROVED** [review]

**Review by wangdayong228 — APPROVED** [review]

File: contracts/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "contracts",
+    "security"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/conflux-fans/conflux-contracts/issues"
+  },
+  "homepage": "https://github.com/conflux-fans/conflux-contracts",
+  "dependencies": {
+    "@openzeppelin/contracts": "4.1.0"
```
Comment (boqiu) [inline contracts/package.json:26]: hardhat dependency missed?

**Review by boqiu — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Pana) [reply package.json:2]: There is a new package.json in contracts folder, which is only used for publishing npm package

File: contracts/package.json (hunk 2 above)
Comment (Pana) [reply contracts/package.json:26]: This is contract's package.json, the outside one still have hardhat dependency

**Review by posaggen — APPROVED** [review]

**Pull request merged**